Code/GraphMol/MolDraw2D/test1.cpp

```diff
@@ -1157,6 +1157,51 @@ void testGithub852() {
     }
     delete m;
   }
+  {
+    std::string smiles =
```

Comment:
As I've been working in the tests, I've slowly been converting them to the modern 'auto m = "CC"_smiles' syntax.